### DISCUSSION TITLE
[Feature Request] Option to easily choose which sites you want to publish to

### DISCUSSION THREAD

**Discussion by Djive** [craftcms/cms#8586] [Ideas]
Would it be possible to have something like "Site Settings" from Sections area (Choose which sites this section should be available in) in Publish area in the right column instead of "Enabled for site"?

The idea is to be able to easily choose which sites you want to publish to, when you are publishing an entry.

"Enabled for site" is ok if you have several sites, but when you have a lot of sites it is a very time consuming task to choose on which sites you want to publish a certain entry.

Thanks!

**Comment by lexislav** [comment]
Especially on multi-site and multi-lang environments, publishing (enabling) entries might be quite complicated and confusing.

Maybe „Enable for Site group“ option (on right publish area) would be very helpful?

It would work (at least) for scenarios like this.

```
Group 1 - company.com
------------------------
company.com
company.com/es
company.com/fr

Group 2 - product-1.com
-------------------------
product-1.com
product-1.com/es
product-1.com/fr
product-1.com/de

etc.
```

and enabling some entry across such sites.
